e2e/container/container_test.go

```diff
@@ -1,57 +1,36 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package e2e
+// Package container runs tests related to container development.
+package container
 
 import (
-	"flag"
-	"os"
-	"path/filepath"
 	"testing"
 
 	"github.com/onsi/ginkgo/v2"
 	"github.com/onsi/gomega"
 	"github.com/runfinch/common-tests/command"
-	"github.com/runfinch/common-tests/option"
 	"github.com/runfinch/common-tests/tests"
-)
 
-const (
-	virtualMachineRootCmd = "vm"
-	installedTestSubject  = "finch"
+	"github.com/runfinch/finch/e2e"
 )
 
-var installed = flag.Bool("installed", false, "the flag to show whether the tests are ran against installed application")
-
-//nolint:paralleltest // TestE2e is like TestMain for our e2e tests.
-func TestE2e(t *testing.T) {
-	description := "Finch CLI e2e Tests"
+//nolint:paralleltest // TestContainer is like TestMain for the container-related tests.
+func TestContainer(t *testing.T) {
```

Comment:
How would we guarantee `TestContainer` runs before `TestVM`?

Reply:
@pendo324's understanding is correct. They will be separated into 2 binaries, so the execution order is guaranteed.

Reply:
Sorry still a bit confused why the execution order is guaranteed? Like `go test ./e2e/container` then `go test ./e2e/vm`

Reply:
> why the execution order is guaranteed?

In `Makefile`:

1. We first run `go test ./e2e/container`, which only compiles and runs the tests defined in the `e2e/container` package, and that means only the container tests are included.
3. We then run `go test ./e2e/vm` to do the same for the VM tests.

Reply:
oh sorry my bad, didn't realize the change in the Makefile.

Reply:
Actually the name makes me think in a way that unit tests are triggered